### ISSUE TITLE
Resampling problem for testing

### ISSUE THREAD

**Issue by haibinswe** [MIC-DKFZ/nnUNet#1390]
Hello fabian,

I am wondering whether resampling is required or optional for the test set in a medical imaging task. Specifically, the spacing of my training set is 0.5x0.5x0.8, but the spacing of the test set is 0.2x0.2x0.2. If I were to resample the test set, would this involve downsampling, thus reducing the image resolution? I'm concerned about introducing bias or artifacts in the model. What are your recommendations.

Additionally, my label represents 3d blood vessels, which are very fine structure and typically have a diameter that is as long as 3-4 voxels. I'm concerned that interpolating the label may distort the blood vessel and result in loss of information. What are your thoughts on this？

**Comment by FabianIsensee** [comment]
Do not, under any circumstances, resample your data. nnU-Net handles everything! (it will resample to your training data resolution)

Interpolating thin structures is and will always be a problem. Your case is difficult because there is such a resolution shift between train and test (why is that? seems like questionable dataset design?). If you notice that this is giving your trouble, you can tell nnU-Net to upsample your training data to the test data distribution by modifying the plans files

**Comment by sten2lu** [comment]
Hi @haibinswe,

Since this issue has already been discussed by Fabian and there has been no recent activity, we will close this issue in 2 weeks (Monday 28th of August).
If this issue happens to persist, please answer in this thread so that we can solve it together.

Best,
Carsten
